api/v1alpha1/capacity_types.go

```diff
@@ -238,13 +238,15 @@ type Quota struct {
 type CapacityStatus struct {
 	// usedResources are the resources (by flavor) currently in use by the
 	// workloads assigned to this capacity
-	UsedResources map[corev1.ResourceName]map[string]Usage `json:"usedResources,omitempty"`
+	UsedResources UsedResources `json:"usedResources,omitempty"`
 
 	// assignedWorkloads is the number of workloads currently assigned to this
 	// capacity.
 	AssignedWorkloads int32 `json:"assignedWorkloads,omitempty"`
 }
 
+type UsedResources map[corev1.ResourceName]map[string]Usage
```

Comment:
Commit says `Add integration test for capacity controller` , but this change doesn't relate to that.
Could you make a commit just for this change, or drop from this PR and move to a tiny PR as a TODO?

Reply:
/lgtm